com.unity.render-pipelines.core/Runtime/Utilities/CoreRenderPipelinePreferences.cs

```diff
@@ -27,26 +36,61 @@ public static Color volumeGizmoColor
             }
         }
 
+        public static Color previewBackgroundColor
```

Comment:
This Preview background color cannot be a User Preference.
See discussion https://unity.slack.com/archives/C0A9NNY9W/p1608024382240300?thread_ts=1607848679.236200&cid=C0A9NNY9W